cpp/dolfinx/mesh/Mesh.h

```diff
@@ -147,4 +147,32 @@ std::tuple<Mesh, std::vector<std::int32_t>, std::vector<std::int32_t>,
 create_submesh(const Mesh& mesh, int dim,
                const std::span<const std::int32_t>& entities);
 
+/// Helper function to get a std::vector of (cell, local_facet) pairs
```

Comment:
Return type isn't a `std::vector`.

Best to avoid using types in the doctstring. The function signature specifies the type.